src/components/TimeButton/TimeButton.js

```diff
@@ -0,0 +1,85 @@
+import React, { useState, useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+import { setTime } from '../../store/Time/time';
+import { get } from '../../utils/snippet';
+
+const Box = styled.div`
+  width: 201px;
+  height: 127px;
+  border-radius: 10px;
+  box-shadow: 0 6px 12px 0 rgba(4, 4, 161, 0.1);
+  border: ${(props) => (props.isClicked ? 'solid 3px #5f5fd9;' : 'none')};
```

Comment:
이부분도 destructuring assignment로 간단하게 만들 수 있을 것 같아요.
 
```suggestion
  border: ${({ isClicked }) => (isClicked ? 'solid 3px #5f5fd9;' : 'none')};
```

Reply:
앗 이부분도 있었네요 감사합니다!